config/langs.toml

```diff
@@ -894,6 +894,32 @@ foreach arg $argv {
 }
 '''
 
+[TeX]
+size    = '9.57 MiB'
+version = '3.141592653'
+website = 'https://tug.org/'
+example = '''
+% Printing
+Hello, World!
+
+% Expanded font
```

Comment:
Drop the expanded font stuff as it won't pass `t/examples.t` with different output to the other langs.